lib/closex/http_client.ex

```diff
@@ -10,22 +10,35 @@ defmodule Closex.HTTPClient do
   @base_url "https://app.close.io/api/v1"
   @behaviour Closex.ClientBehaviour
 
-  # Leads
-
   ## TODO: httpoison opts should move underneath the `:httpoison` key
 
   @doc "List or search for leads: https://developer.close.io/#leads-list-or-search-for-leads"
   def find_leads(search_term, opts \\ []) do
+    opts = merge_search_term_into_opts(search_term, opts)
+
+    # HACK: Workaround bug in HTTPoison, see: https://github.com/edgurgel/httpoison/issues/285
```

Comment:
Great call on the comment, what's the workaround though? Can you point to it in the comment please, e.g.

> Pass an empty struct as third argument to avoid bug in HTTPoison